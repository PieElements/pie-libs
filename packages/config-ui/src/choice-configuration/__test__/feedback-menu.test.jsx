import React from 'react';
import FeedbackMenu from '../feedback-menu';
import renderer from 'react-test-renderer';

describe('feedback-menu', () => {

  it('renders correctly', () => {
    const tree = renderer
      .create(<FeedbackMenu/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
