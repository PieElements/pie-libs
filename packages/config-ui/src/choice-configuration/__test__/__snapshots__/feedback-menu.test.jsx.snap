// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`feedback-menu renders correctly 1`] = `
<div>
  <div
    onClick={[Function]}
  />
  <div>
    Menu
  </div>
</div>
`;
